WEB_SERVICE/WEB/_HIGH/FIELD_DUPLICATION/meta.json

```diff
@@ -0,0 +1,37 @@
+{
+    "risk_rating": "high",
+    "title": "Field Duplication in GraphQL API",
+    "short_description": "Exploiting a GraphQL server by duplicating fields, potentially leading to Denial of Service attacks.",
+    "references": {
+      "imperva": "https://www.imperva.com/blog/graphql-vulnerabilities-common-attacks/",
+      "escape": "https://docs.escape.tech/testing/vulnerabilities/resource_limitation/graphql_field_duplication",
```

Comment:
Please avoid both.

Reply:
Prefer resources by OWASP and non-company orgs.

Reply:
fixed!